### ISSUE TITLE
Improve S5122: Support cors middleware

### ISSUE THREAD

**Issue by saberduck** [SonarSource/SonarJS#2058]
### Add support for Express.js cors middleware 

https://expressjs.com/en/resources/middleware/cors.html

Code examples: https://github.com/SonarSource/security-expected-issues/tree/master/javascript/rules/hotspots/RSPEC-5122

For the `cors` middleware raise an issue when the `origin` property of the options is set to `*`:
```
let corsOptions = {
  origin: '*' // Sensitive (main location)
};

let app = express();
app.use(cors(corsOptions));  // Sensitive (secondary location on the cors() call)
```
Without options passed in the call (ie: `cors()`) the code is sensitive (equals to `origin` set to `*`)

Related issue:
https://github.com/SonarSource/SonarJS/issues/1145
